lib/image-inspector-client.rb

```diff
@@ -51,12 +51,7 @@ def fetch_metadata
     def exception_handler
       yield
     rescue RestClient::Exception => e
-      begin
-        json_error_msg = JSON.parse(e.response || '') || {}
-      rescue JSON::ParserError
-        json_error_msg = {}
-      end
-      err_message = json_error_msg['message'] || e.to_s
+      err_message = e.response || e.message || ''
```

Comment:
@moolitayer why don't you fallback to this only when `JSON::ParserError` is thrown... so in the future we can fix image-inspector to return json and this code will be already compatible.